.github/CODEOWNERS

```diff
@@ -28,6 +28,7 @@ exporter/jaegerthrifthttpexporter/         @open-telemetry/collector-contrib-app
 exporter/kinesisexporter/                  @open-telemetry/collector-contrib-approvers @owais
 exporter/loadbalancingexporter/            @open-telemetry/collector-contrib-approvers @jpkrohling
 exporter/logzioexporter/                   @open-telemetry/collector-contrib-approvers @yyyogev
+exporter/lokiexporter/                     @open-telemetry/collector-contrib-approvers @gramidt
```

Comment:
Would be good to have 2 people so if you create a PR the other person reviews initially the PR.

Reply:
Absolutely! I'll ask around who would like to join me as the owner for this exporter. If you have anyone in mind, let me know. 

Thank you again for all of your help!